README.md

```diff
@@ -1,5 +1,21 @@
+[Dynatrace](https://www.dynatrace.com/integrations/opentelemetry) supports native
+OpenTelemetry protocol (OTLP) ingest for traces, metrics and logs.
+All signals can be sent directly to Dynatrace via **OTLP protobuf over HTTP**
+using the built-in OTLP/HTTP Exporter available in the OpenTelemetry .NET SDK.
+More information on configuring your .NET applications to use the OTLP exporter can be found in the
+[Dynatrace documentation](https://www.dynatrace.com/support/help/shortlink/otel-wt-dotnet#tabgroup--dynatrace-docs--otlp-export).
```

Comment:
```suggestion
[Dynatrace documentation](https://www.dynatrace.com/support/help/shortlink/otel-wt-dotnet).
```
The anchor could be omitted.

Reply:
I thought about going to the anchor to see directly the OTLP configuration steps. There's a lot in the page so maybe that saves them to look around?